pkg/tagger/collectors/workloadmeta_extract.go

```diff
@@ -137,6 +137,8 @@ func (c *WorkloadMetaCollector) processEvents(evBundle workloadmeta.EventBundle)
 				tagInfos = append(tagInfos, c.handleECSTask(ev)...)
 			case workloadmeta.KindContainerImageMetadata:
 				tagInfos = append(tagInfos, c.handleContainerImage(ev)...)
+			case workloadmeta.KindProcess:
+				tagInfos = append(tagInfos, c.handleProcess(ev)...)
```

Comment:
Should we skip enriching the tagger with process data for now, similarly to what has been done with ContainerImages ? This may better fit a different PR with dedicated unit tests for this code path. 

https://github.com/DataDog/datadog-agent/commit/73ef8f4f77e320493d4bf8168cf08b88f765f829#diff-47f72951df75b2af60591fb7810afca15eb0ef474e83928334fbc32bf29f8eceR136-R137

Reply:
Should be fixed by now